AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/CascadeDeleteOperation.swift

```diff
@@ -142,8 +146,19 @@ public class CascadeDeleteOperation<M: Model>: AsynchronousOperation {
         
         let modelIds = queriedModels.map { $0.identifier(schema: self.modelSchema).stringValue }
         
-        associatedModels = await self.recurseQueryAssociatedModels(modelSchema: self.modelSchema, ids: modelIds)
+        if modelName != "MutationEvent" {
+            self.log.debug("[CascadeDelete.1] Deleting \(modelName) with identifiers: \(modelIds)")
+        }
         
+        associatedModels = await self.recurseQueryAssociatedModels(modelSchema: self.modelSchema, ids: modelIds)
+        if self.log.logLevel >= .debug {
+            if modelName != "MutationEvent" {
+                self.log.debug("[CascadeDelete.2] Queried for \(modelName) associated models, found \(associatedModels.compactMap { $1.modelName })")
+                
+                self.log.debug("[CascadeDelete.2] [Query ids of associated Models] Queried for \(associatedModels.compactMap({$1.modelName})), retrieved ids for deletion: \(associatedModels.map { $0.1.identifier(schema: modelSchema).stringValue })")
```

Comment:
see above comment about taking a look at `recurseQueryAssociatedModels` to see if we can do this logging during the recursive query logic